using-valtimo/process/systemprocesses.md

```diff
@@ -0,0 +1,31 @@
+# System processes
+
+System processes are a special kind of process. A system process describes an often used process 
```

Comment:
I would avoid fluff. Rather than stating they're a special kind of process, describe what they are right away. Maybe rephrase to:

System processes are processes that are critical to the functioning of Valtimo itself. Since these processes should be handled with care, these processes are flagged as read-only by default. When a process is read-only, it is not possible to update the process or deploy it.